WatchFaceFormat/SimpleDigital/res/raw/watchface.xml

```diff
@@ -49,7 +49,7 @@
         <Text align="CENTER">
           <!-- Demonstrates using Android string resources for localization -->
           <Font family="SYNC_TO_DEVICE" size="36" color="#ffffffff">
-            <Template>@string/greeting</Template>
+            <Template>%s<Parameter expression="greeting"></Parameter></Template>
```

Comment:
Not really about this PR, but I can't follow that this is correct from

https://developer.android.com/training/wearables/wff/group/part/text/formatter/template
https://developer.android.com/training/wearables/wff/common/attributes/arithmetic-expression

Reply:
I'll verify the snippets on those pages work correctly or not, and update them to cover the above. Thanks Yuri!

Reply:
I guess It's more a question of why

`<Parameter expression="greeting"></Parameter>` evaluates to R.string.greeting

While https://developer.android.com/training/wearables/wff/group/part/text/formatter/template shows

```
<Template>%s %d
    <Parameter expression="Current heart rate: " />
    <Parameter expression="[HEART_RATE]" />
</Template>
```